### ISSUE TITLE
Build for SQS not working ?

### ISSUE THREAD

**Issue by sebgoa** [aws/aws-controllers-k8s#303]
**Describe the bug**
First attempt at the docker build for SQS service fails:

```
make build-controller-image

...
 => => transferring context: 180.36MB                                                                                                                  25.1s
 => [builder 2/7] WORKDIR /github.com/aws/aws-controllers-k8s                                                                                           0.7s
 => [builder 3/7] COPY go.mod go.mod                                                                                                                    0.1s
 => [builder 4/7] COPY go.sum go.sum                                                                                                                    0.1s
 => [builder 5/7] RUN  go mod download                                                                                                                203.2s
 => [builder 6/7] COPY . /github.com/aws/aws-controllers-k8s/                                                                                           4.4s
 => ERROR [builder 7/7] RUN  go build -a -o /github.com/aws/aws-controllers-k8s/bin/controller /github.com/aws/aws-controllers-k8s/services/sqs/cmd/c  85.1s
------
 > [builder 7/7] RUN  go build -a -o /github.com/aws/aws-controllers-k8s/bin/controller /github.com/aws/aws-controllers-k8s/services/sqs/cmd/controller/main.go:
#13 84.69 # github.com/aws/aws-controllers-k8s/services/sqs/pkg/resource/queue
#13 84.69 services/sqs/pkg/resource/queue/sdk.go:49:7: rm.requiredStatusFieldsMissingFromGetAttributesInput undefined (type *resourceManager has no field or method requiredStatusFieldsMissingFromGetAttributesInput)
------
failed to solve with frontend dockerfile.v0: failed to build LLB: executor failed running [/bin/sh -c go build -a -o $work_dir/bin/controller $work_dir/services/$service_alias/cmd/controller/main.go]: runc did not terminate sucessfully
make: *** [build-controller-image] Error 2
```

**Expected outcome**
it works :)

**Environment**

```
$ go version
go version go1.15.2 darwin/amd64
(⎈ |k3d-foo:default)sebair:aws-controllers-k8s sebgoa$ docker version
Client: Docker Engine - Community
 Version:           19.03.12
 API version:       1.40
 Go version:        go1.13.10
 Git commit:        48a66213fe
 Built:             Mon Jun 22 15:41:33 2020
 OS/Arch:           darwin/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.12
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.10
  Git commit:       48a66213fe
  Built:            Mon Jun 22 15:49:27 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.2.13
  GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

**Comment by jaypipes** [comment]
Hi @sebgoa! Yeah, I'm working on the fix for this here: #296. Should have a fix pushed within a few hours. Sorry for the delay!

**Comment by jaypipes** [comment]
The build for the SQS service controller should now be working. I cannot reproduce the build failure any more:

```
jaypipes@thelio:~/go/src/github.com/aws/aws-controllers-k8s$ make build-controller-image SERVICE=sqs
building 'sqs' controller docker image with tag: ack-sqs-controller:aeed258-dirty
[+] Building 15.7s (15/15) FINISHED                                                                                                                                                                                
 => [internal] load build definition from Dockerfile                                                                                                                                                          0.0s
 => => transferring dockerfile: 38B                                                                                                                                                                           0.0s
 => [internal] load .dockerignore                                                                                                                                                                             0.0s
 => => transferring context: 2B                                                                                                                                                                               0.0s
 => [internal] load metadata for docker.io/library/amazonlinux:2                                                                                                                                              0.6s
 => [internal] load metadata for docker.io/library/golang:1.14.1                                                                                                                                              0.7s
 => [builder 1/7] FROM docker.io/library/golang:1.14.1@sha256:38a520b05b971181a5a30d72b74d1b70c63688eb267a4fb376c7476a5c6f885b                                                                                0.0s
 => [internal] load build context                                                                                                                                                                             0.2s
 => => transferring context: 535.13kB                                                                                                                                                                         0.2s
 => CACHED [stage-1 1/3] FROM docker.io/library/amazonlinux:2@sha256:2c99363fc74d3a39f02365b964e73cceb2b2524c00e9977e16680156e2f79ee8                                                                         0.0s
 => CACHED [builder 2/7] WORKDIR /github.com/aws/aws-controllers-k8s                                                                                                                                          0.0s
 => CACHED [builder 3/7] COPY go.mod go.mod                                                                                                                                                                   0.0s
 => CACHED [builder 4/7] COPY go.sum go.sum                                                                                                                                                                   0.0s
 => CACHED [builder 5/7] RUN  go mod download                                                                                                                                                                 0.0s
 => [builder 6/7] COPY . /github.com/aws/aws-controllers-k8s/                                                                                                                                                 0.7s
 => [builder 7/7] RUN GIT_VERSION=$(git describe --tags --dirty --always) &&     GIT_COMMIT=$(git rev-parse HEAD) &&     BUILD_DATE=$(date +%Y-%m-%dT%H:%M) &&     go build -ldflags="-X github.com/aws/aws  13.6s
 => [stage-1 2/3] COPY --from=builder /github.com/aws/aws-controllers-k8s/bin/controller /github.com/aws/aws-controllers-k8s/LICENSE /github.com/aws/aws-controllers-k8s/ATTRIBUTION.md /bin/                 0.1s
 => exporting to image                                                                                                                                                                                        0.2s
 => => exporting layers                                                                                                                                                                                       0.2s
 => => writing image sha256:875912bd5921e935434feb515b179608d54597ab1eb077debecd51ca9a4a418d                                                                                                                  0.0s
 => => naming to docker.io/library/ack-sqs-controller:aeed258-dirty    
```
